docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md

```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
+
+| Flag | Description | Default |
+| :--- | :---------- | :------ |
+| --data_dir | Data directory to be backed up. | "/opt/yugabyte" |
+| &#8209;&#8209;disable_version_check | Exclude version metadata when creating backup. | false |
+| --exclude_prometheus | Exclude Prometheus metric data from backup. | false |
+| --exclude_releases | Exclude YugabyteDB releases from backup. | false |
+| -h, --help | Help for `createBackup`. | |
+| --skip_restart | Don't restart processes during execution. | true |
+| --verbose | Display extra information in the output. | false |
+| -f, --force (global flag) | Run in non-interactive mode. All user confirmations are skipped. | |
+| --log_level (global flag) | Log level for this command.<br>Levels: panic, fatal, error, warn, info, debug, trace. | "info" |
+
 ## Restore YugabyteDB Anywhere
 
 To restore the YugabyteDB Anywhere content from your saved backup, perform the following:
 
-1. Copy the backup `.tar` file from your storage location.
+1. Copy the backup file from your storage location.
 
 1. Run the `restoreBackup` command as follows:
 
     ```sh
     sudo yba-ctl restoreBackup <input_path> [flags]
     ```
 
-    The `restoreBackup` command executes the `yb_platform_backup.sh` script to restore from a previously taken backup of your YugabyteDB Anywhere instance. Specify the `input_path` to the backup `.tar.gz` file as the only argument.
+    Specify the `input_path` to the backup file as the only argument. For example:
+
+    ```sh
+    $ sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
+    ```
+
+The `restoreBackup` command executes the `yb_platform_backup.sh` script to restore from the specified backup of your YugabyteDB Anywhere instance.
 
 When finished, the restored YugabyteDB Anywhere is ready to continue managing your universes and clusters.
 
+### Restore options
+
 The following table describes optional flags you can include with the `restoreBackup` command.
 
-| Flag | Description |
-| :--- | :---------- |
-| --destination string | Path to un-tar the backup. (default: "/opt/yugabyte") |
-| -h, --help | Help for restoreBackup. |
-| --migration | Restoring from a Replicated or Yugabundle installation. (default: false) |
-| --skip_dbdrop | Skip dropping the yugaware database before a migration restore. (default: false) |
-| --skip_restart | Don't restart processes during command execution. (default: false) |
-| --use_system_pg | Use system path's `pg_restore` as opposed to installed binary. (default: false) |
-| --verbose | Display extra information in the output. (default: false) |
-| -f, --force (global flag) | Run in non-interactive mode. All user confirmations are skipped. |
-| --log_level string (global flag) | Log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default: "info") |
\ No newline at end of file
+| Flag | Description | Default |
+| :--- | :---------- | :------ |
+| --destination | Path to un-tar the backup. | "/opt/yugabyte" |
```

Comment:
Same with this yba-ctl. The default also isn't necessarily /opt/yugabyte but whatever they set for installRoot. We should probably just let yba-ctl figure this out it's more likely to go wrong if customer specifies this